BridgeAppSDK/SBAProfileItem.swift

```diff
@@ -708,6 +709,366 @@ open class SBAStudyParticipantCustomAttributesProfileItem: SBAProfileItemBase {
     }
 }
 
+open class SBAWhatAndWhen: NSObject, Comparable {
+    static var valueKey: String { return #keyPath(value) }
+    static var dateKey: String { return #keyPath(date) }
+    static var isNewKey: String { return #keyPath(isNew) }
+    open var value: SBBJSONValue
+    open var date: NSDate
+    var isNew: Bool
```

Comment:
`isChanged`? Is this flag only set if the item is new or also if it needs to be updated on the server?